http/vibe/http/fileserver.d

```diff
@@ -331,29 +331,27 @@ private void sendFileImpl(scope HTTPServerRequest req, scope HTTPServerResponse
 		try {
 			if (s[0].length) {
 				rangeStart = s[0].to!ulong;
-				rangeEnd = s[1].length ? s[1].to!ulong : dirent.size;
+				rangeEnd = s[1].length ? s[1].to!ulong : dirent.size-1;
```

Comment:
could this overflow with dirent.size == 0?

Reply:
You are right, I had singed types in mind, when writing this. I actually first changed `rangeEnd` to be exclusive, which didn't have this issue, but decided against it to keep the diff smaller, but I guess that's nicer than changing everything to signed.